app/models/post.rb

```diff
@@ -10,4 +10,27 @@ def comments_counter
   def likes_counter
     likes.count
   end
+
+  def update_posts_counter
+    author.update(posts_count: author.posts_count + 1)
+  end
+
+  def update_likes_counter
+    likes_count = likes.count
+    update(likes_count:)
+  end
```

Comment:
- [ ] Please implement the following method in the `Like` model as the method is used whenever a `Like` is created not a `Post`, as per [this](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Processing_data_model_project_v1_1.md#:~:text=for%20a%20post.-,Likes,A%20method%20that%20updates%20the%20likes%20counter%20for%20a%20post.,-Go%20to%20rails) requirement.